src/App.js

```diff
@@ -246,100 +250,63 @@ function Dashboard({statuses, version}: DashboardPropType) {
       </p>
     );
   } else {
-    const {
-      archive,
-      product_details,
-      release_notes,
-      security_advisories,
-      download_links,
-    } = statuses;
     return (
       <div>
-        <table className="table">
-          <tbody>
-            <tr>
-              <td>
-                <h2>Archives</h2>
-                <DisplayStatus
-                  url={
-                    'https://archive.mozilla.org/pub/firefox/releases/' +
-                    version +
-                    '/'
-                  }
-                  data={archive}
-                />
-              </td>
-
-              <td>
-                <h2>Product Details</h2>
-                <DisplayStatus
-                  url={'https://product-details.mozilla.org/1.0/firefox.json'}
-                  data={product_details}
-                />
-              </td>
-            </tr>
-
-            <tr>
-              <td>
-                <h2>Release Notes</h2>
-                <DisplayStatus
-                  url={
-                    'https://www.mozilla.org/en-US/firefox/' +
-                    version +
-                    '/releasenotes/'
-                  }
-                  data={release_notes}
-                />
-              </td>
-
-              <td>
-                <h2>Security Advisories</h2>
-                <DisplayStatus
-                  url={
-                    'https://www.mozilla.org/en-US/security/known-vulnerabilities/firefox/'
-                  }
-                  data={security_advisories}
-                />
-              </td>
-
-              <td>
-                <h2>Download links</h2>
-                <DisplayStatus
-                  url={'https://www.mozilla.org/en-US/firefox/all/'}
-                  data={download_links}
-                />
-              </td>
-            </tr>
-          </tbody>
-        </table>
+        <h2>
+          Channel: {(releaseInfo && releaseInfo.channel) || ''}
+        </h2>
+        <div className="dashboard">
+          {Object.keys(checkResults).map(key =>
```

Comment:
We should make sure to display the checks in the order they are in the endpoint listing them so that we can create a timeline and see were it was stuck in the release process.